napari_process_points_and_surfaces/__init__.py

```diff
@@ -1,5 +1,5 @@
 
-__version__ = "0.1.2"
+__version__ = "0.1.3"
```

Comment:
```suggestion
__version__ = "0.1.2"
```
Leave version bumps to the maintainer ;-)